MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile

```diff
@@ -5,4 +5,5 @@ target 'PodInstall' do
   # The branch of MapboxNavigation and MapboxNavigation will be replaced by Bitrise to use the current branch.
   pod 'MapboxNavigation', :path => '../../../'
   pod 'MapboxCoreNavigation', :path => '../../../'
+  pod 'MapboxDirections.swift', :git => 'git@github.com:mapbox/MapboxDirections.swift.git', :branch => 'return-response'
```

Comment:
Remember to remove this explicit dependency once we’re back on an official release.